scripts/v8-to-v9.py

```
@@ -0,0 +1,178 @@
+    for validator in blank_data['app_state']['dogfood']['val_set']:
+        last_total_power += int(validator['power'])
+    blank_data['app_state']['dogfood']['last_total_power'] = str(last_total_power)
+
+    # x/oracle -> copy
+    blank_data['app_state']['oracle']['params'] = data['app_state']['oracle']['params']
+    ## set a default value for the new field
+    blank_data['app_state']['oracle']['params']['piece_size_byte'] = "48000"
+    ## todo: there might be other necessary operations for the oracle module 
+    blank_data['app_state']['oracle']['params']['rules'] = [
+        {},
+        {"source_ids":["0"]},
```

Flake8 (7.2.0)

[error] missing whitespace after ':'

(E231)